os/android/iahwc2.cpp

```diff
@@ -815,6 +855,25 @@ HWC2::Error IAHWC2::HwcDisplay::SetActiveConfig(hwc2_config_t config) {
   return HWC2::Error::None;
 }
 
+HWC2::Error IAHWC2::HwcDisplay::SetActiveConfigWithConstraints(
+  hwc2_config_t config,
+  hwc_vsync_period_change_constraints_t* vsyncPeriodChangeConstraints,
+  hwc_vsync_period_change_timeline_t* outTimeline
+  ) {
+  supported(__func__);
+  return SetActiveConfig(config);
+}
+
+HWC2::Error IAHWC2::HwcDisplay::GetDisplayVsyncPeriod(hwc2_vsync_period_t* outVsyncPeriod) {
+  supported(__func__);
+  *outVsyncPeriod = 16666667;
```

Comment:
Use a Macro like DEFAULT_VSYNC_PERIOD defined in head file

Reply:
why the vsync period need to be initialized with default here?
no need to do that, refer to https://github.com/intel/IA-Hardware-Composer/blob/master/wsi/drm/drmdisplay.cpp#L273